test/circuit_optimizer/test_collect_blocks.cpp

```diff
@@ -16,15 +16,31 @@
 namespace qc {
 TEST(CollectBlocks, emptyCircuit) {
   QuantumComputation qc(1);
-  qc::CircuitOptimizer::collectBlocks(qc, 1);
+  qc::CircuitOptimizer::collectBlocks(qc, 1, false);
+  EXPECT_EQ(qc.getNindividualOps(), 0);
+}
+
+TEST(CollectBlocks, emptyCliffordCircuit) {
+  QuantumComputation qc(1);
+  qc::CircuitOptimizer::collectBlocks(qc, 1, true);
```

Comment:
This is a pretty useless test. The new flag has no influence at all on empty circuits.